Cpp_hands_on_programming/lib/queue/Ipc_queue.cc

```diff
@@ -9,103 +9,103 @@ void Ipc_queue::send()
 {
     std::cout << "Starting queueSend.." << std::endl;
     
-    // check_leadingSlash();
+    check_Slash();
```

Comment:
I wonder if this check shouldn't be rather done in constructor. There is no use of this class instance if pipe name is wrong

Reply:
That makes sense, I never really thought about throwing exception in constructor before, seems like a logical way to do it. Thx!